pkg/scheduler/framework/interface.go

```diff
@@ -106,45 +106,40 @@ const (
 // the reasons should explain why.
 // NOTE: A nil Status is also considered as Success.
 type Status struct {
-	code    Code
-	reasons []string
-	err     error
+	Code    Code
+	Reasons []string
+	Err     error
 }
 
 // Code returns code of the Status.
-func (s *Status) Code() Code {
+func (s *Status) GetCode() Code {
```

Comment:
I am not confident about this one. If we export `code`, this method is still needed to hide the details that nil is a Success code. But we cannot enforce callers to use this method instead of using exported `Code`.